packages/@uppy/drag-drop/src/index.js

```diff
@@ -105,6 +105,10 @@ module.exports = class DragDrop extends Plugin {
         // Nothing, restriction errors handled in Core
       }
     })
+
+    // ___Why not use value="" on <input/> instead?
+    //    Because if we use that method of clearing the input, Chrome will not trigger onChange={} if we drop the same file twice (Issue #768).
+    event.target.value = null
```

Comment:
We pass `ev` to the callback, but `event` works anyway. What sort of sorcery is this? 🙀 Been trying to find documentation for a while, nothing so far.

Reply:
It's a global property: https://developer.mozilla.org/en-US/docs/Web/API/window/event

It allows you to refer to the event in an inline `on*` event handler attribute, eg `<input onclick="event.target.value = undefined">`

best not to rely on it :)

Reply:
Wow, thanks! Crazy. A thought about it being global occurred to me briefly when eslint didn’t complain.

Is it safe to call the callback argument `event`: `(event) => { ... }`? Thus overriding this global `event`.

Reply:
I didn't fucking notice! It wasn't intentional, I'm amazed it worked too.
I'm used to things like this being caught by eslint, we seem to be having a very lenient configuration? I also noticed it doesn't tell me what class methods are not used.

Reply:
> Is it safe to call the callback argument `event`: `(event) => { ... }`? Thus overriding this global `event`.

yes

i'm also surprised that this isn't caught, I thought standard has a `no-confusing-globals` rule or something, may need to look into that

Reply:
Standard includes this http://eslint.org/docs/rules/no-undef, couldn’t figure out why that wouldn’t work yet.

There is another rule we could add, it’s talking about `event` specifically: https://eslint.org/docs/rules/no-restricted-globals